README.md

```diff
@@ -1,5 +1,9 @@
 ﻿# Z-Wave Home Automation library for .NET
 
+## About this fork
+I've forked this to remove NLog as the project I am using this library in doesn't use NLog thus adding more libraries seem useless.
+This fork also contains WIP's for adding new commandclasses to the library when I stumble upon one that's missing.
+
```

Comment:
@bluewalk I think you create a new branch. Just add the *CentralScene* command class there and submit the PR. If you need other specific customizations that are not supposed to be merged in the master repo you can create other branches and sync with master whenever there are updates from the upstream.